modules/history/src/views/full/conversation.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import style from './style.scss'
+
+import 'react-day-picker/lib/style.css'
+
+import { TiRefresh } from 'react-icons/ti'
+import { FaFilter } from 'react-icons/fa'
+import DayPickerInput from 'react-day-picker/DayPickerInput'
+
+function QueryOptions(props) {
+  return (
+    <div className={style['query-options']}>
+      <div className={style['query-options-daypick']}>
+        <div className={style['query-options-from_to']}>from:</div>
+        <div className={style['daypicker-item']}>
+          <DayPickerInput value={props.defaultFrom} onDayChange={props.handleFromChange} />
+        </div>
+      </div>
+      <div className={style['query-options-daypick']}>
+        <div className={style['query-options-from_to']}>to:</div>
+        <div className={style['daypicker-item']}>
+          <DayPickerInput value={props.defaultTo} onDayChange={props.handleToChange} />
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export class ConversationPicker extends React.Component {
+  state = {
+    displayQueryOptions: false
+  }
+
+  toggleFilters() {
+    this.setState({ displayQueryOptions: !this.state.displayQueryOptions })
+  }
+
+  render() {
+    return (
+      <div className={style['conversations']}>
+        <div className={style['conversations-titlebar']}>
+          <div>Conversations</div>
+          <div className={style['conversations-icons']}>
+            <FaFilter className={style['conversations-filter']} onClick={this.toggleFilters.bind(this)} />
```

Comment:
**that's perfectly OK** but JFYI you can also declare `toggleFilters()` as an arrow fn `toggleFilters = () =>` and it will be bound to the class instance automatically